frontend/packages/dev-console/src/components/GuidedTourTile.scss

```diff
@@ -0,0 +1,16 @@
+.odc-guidedtour-tile {
+  &__card {
+    height: 100%;
```

Comment:
```
  height: $co-m-catalog-tile-height;
  width: $co-m-catalog-tile-width;
```

Reply:
The above aligns with the other empty state tile sizes.

Reply:
updated. boils down to 100% height though :)